implementations/CMIP6-UofT/add_CMIP6.py

```diff
@@ -90,7 +91,8 @@ class STACItem(BaseModel):
 def make_cmip6_id(attrs: MutableMapping[str, Any]) -> str:
     """Return unique ID for CMIP6 data collection (multiple variables)."""
     keys = ["activity_id", "institution_id", "source_id", "experiment_id", "variant_label", "table_id", "grid_label",]
-    return "_".join(attrs[k] for k in keys)
+    item_name = "_".join(attrs[k] for k in keys)
```

Comment:
I've made a change to this function in #16. I included `variable_id` in the list so that the entire string becomes unique. I believe that string is what ESGF uses to uniquely identify each dataset. My thinking was that we could use the same string for the ID here. Is there an advantage to using a hash rather than a human readable string?